webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+const TerserPlugin = require('terser-webpack-plugin');
+
+module.exports = {
+  mode: 'production',
+  entry: ['./src/index.js'],
+  output: {
+    path: path.join(process.cwd(), 'umd'),
+    filename: 'index.js',
+    sourceMapFilename: 'index.js.map',
+    library: '@makerdao/dai-plugin-governance',
```

Comment:
i hope we can avoid duplicating the whole file by passing this in as an environment variable or something

Reply:
yeah, agreed -- should be possible. note that this PR is still in cleanup mode, will add reviewers once it's ready :)